patches/extra/bromite/flag-fingerprinting-canvas-image-data-noise.patch

```diff
@@ -65,29 +65,29 @@ approach to change color components.
    BLINK_PLATFORM_EXPORT static void EnableSignedExchangeSubresourcePrefetch(
 --- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
 +++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
-@@ -21,6 +21,7 @@
+@@ -24,6 +24,7 @@
  #include "third_party/blink/renderer/platform/geometry/float_quad.h"
  #include "third_party/blink/renderer/platform/graphics/bitmap_image.h"
  #include "third_party/blink/renderer/platform/graphics/skia/skia_utils.h"
 +#include "third_party/blink/renderer/platform/graphics/static_bitmap_image.h"
  #include "third_party/blink/renderer/platform/graphics/stroke_data.h"
  #include "third_party/blink/renderer/platform/heap/heap.h"
  
-@@ -2005,6 +2006,11 @@ ImageData* BaseRenderingContext2D::getIm
+@@ -2180,6 +2181,11 @@ ImageData* BaseRenderingContext2D::getIm
            snapshot->PaintImageForCurrentFrame().GetSkImageInfo().bounds();
        DCHECK(!bounds.intersect(SkIRect::MakeXYWH(sx, sy, sw, sh)));
      }
 +    if (read_pixels_successful && RuntimeEnabledFeatures::FingerprintingCanvasImageDataNoiseEnabled()) {
-+      base::span<uint8_t> dst = base::span<uint8_t>(reinterpret_cast<uint8_t*>(contents.Data()),
-+          contents.DataLength());
-+      StaticBitmapImage::ShuffleSubchannelColorData(dst.data(), image_info, sx, sy);
++      base::span<uint8_t> dst = base::span<uint8_t>(reinterpret_cast<uint8_t*>(image_data_pixmap.writable_addr()),
++          image_data_pixmap.rowBytes());
++      StaticBitmapImage::ShuffleSubchannelColorData(dst.data(), image_data_pixmap.info(), sx, sy);
```

Comment:
`Multiple-fingerprinting-mitigations.patch` from Bromite 92 has a simpler way of handling this.